tests/controller_tests/test_retrieve.py

```diff
@@ -1,4 +1,35 @@
+import io
+
+import pytest
+
+from config import Config
+from core.aws import _S3_CLIENT
 from core.const import EXCEL_MIMETYPE
+from tests.integration_tests.test_aws import create_bucket, delete_bucket
+
+
+@pytest.fixture(autouse=True, scope="module")
+def test_buckets():
+    """Sets up and tears down buckets used by this module.
+
+    On set up:
+    - creates data-store-successful-files-unit-tests
+
+    On tear down, deletes all objects stored in the buckets and then the buckets themselves.
+    """
+    create_bucket(Config.AWS_S3_BUCKET_SUCCESSFUL_FILES)
+    yield
+    delete_bucket(Config.AWS_S3_BUCKET_SUCCESSFUL_FILES)
+
+
+@pytest.fixture()
+def uploaded_mock_file():
+    """Uploads a mock generic file and deletes it on tear down."""
+    fake_file = io.BytesIO(b"0x01010101")
+    fake_filename = "S-R03-1"
+    _S3_CLIENT.upload_fileobj(fake_file, Config.AWS_S3_BUCKET_SUCCESSFUL_FILES, fake_filename)
+    yield
+    _S3_CLIENT.delete_object(Bucket=Config.AWS_S3_BUCKET_SUCCESSFUL_FILES, Key=fake_filename)
```

Comment:
We have a similar fixture in `test_aws.py` - we should be able to pull out most of the logic from both and reuse it
https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/197952210b27ddfc87ea5343409d5f30fdb4098f/tests/integration_tests/test_aws.py#L58-L65